engine/src/main/java/org/terasology/engine/rendering/dag/RenderGraph.java

```diff
@@ -149,7 +149,8 @@ private void connect(Node... nodeList) {
                 if (!graph.hasEdgeConnecting(fromNode, toNode)) {
                     graph.putEdge(fromNode, toNode);
                 } else {
-                    logger.warn("Trying to connect two already connected nodes, {} and {}", fromNode.getUri(), toNode.getUri());
+                    logger.atWarn().addArgument(fromNode.getUri()).addArgument(toNode.getUri()).
+                            log("Trying to connect two already connected nodes, {} and {}", fromNode.getUri(), toNode.getUri());
```

Comment:
simple getter methods
```suggestion
                    logger.warn("Trying to connect two already connected nodes, {} and {}", fromNode.getUri(), toNode.getUri()); //NOPMD
```